plugin.json

```
@@ -29,11 +29,20 @@
         },
         {
           "key": "EncryptionKey",
-          "display_name": "Encryption Key:",
+          "display_name": "At Rest Encryption Key:",
           "type": "generated",
-          "help_text": "The encryption key used to encrypt the authorization token.",
-          "regenerate_help_text": "Regenerates the encryption key for encrypting the authorization token. Regenerating the authorization invalidates your existing Confluence connection."
-      }
+          "help_text": "The encryption key used to encrypt stored API tokens.",
+          "placeholder": "",
+          "default": null,
+          "secret": true
+        },
+        {
+          "key": "AdminAPIToken",
+          "display_name": "Admin API Token",
```

Would this make it more clear?
```suggestion
          "display_name": "Confluence Admin API Token",
```